ee/tabby-db/src/users.rs

```diff
@@ -208,16 +208,17 @@ impl DbConn {
     }
 
     pub async fn update_user_active(&self, id: i32, active: bool) -> Result<()> {
```

Comment:
please also add a unit test on this

Reply:
(expecting error if there's no change)

Reply:
I was pushing the test as you left this comment :')